### PR TITLE
Fix for Github.com

### PR DESCRIPTION
**Pull request by BananaManCJ**

### CODE REVIEW

**Comment by alexanderby** [comment]
Hi, could you please explain, what have changed?

**Comment by BananaManCJ** [comment]
Before: 
image
After: 
image

**Comment by BananaManCJ** [comment]
It's not perfect, but it looks better

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -935,6 +928,9 @@ CSS
     background-color: rgb(28, 30, 31) !important;
     color: darkgrey !important;
 }
+.jfk-bubble, .octotree-sidebar, .cc-pr__logo, .cc-octicon, #network canvas, img.network-tree {
+    filter: invert(94%) hue-rotate(180deg) contrast(90%) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config]: Looks' better

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: invert(94.4%)

Comment (BananaManCJ) [reply src/config/dynamic-theme-fixes.config]: That looks way better, nice!

Comment (BananaManCJ) [reply src/config/dynamic-theme-fixes.config]: Idk how to put that into my pull request

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: Go to changes then on the 3 dots of on right corner of the 'file' and then edit file

Comment (Mertyn) [reply src/config/dynamic-theme-fixes.config]: That's basically what i did initially, when i fixed the issue, but @alexanderby just changed it to the INVERT section :D

**Pull request merged**